posthog/hogql_queries/error_tracking_query_runner.py

```diff
@@ -504,13 +501,12 @@ def error_tracking_issues(self, ids):
             }
 
             assignment_user_id = issue.get("assignment__user_id")
-            assignment_user_group_id = issue.get("assignment__user_group_id")
             assignment_role_id = issue.get("assignment__role_id")
 
-            if assignment_user_id or assignment_user_group_id or assignment_role_id:
+            if assignment_user_id or assignment_role_id:
                 result["assignee"] = {
-                    "id": assignment_user_id or str(assignment_user_group_id) or str(assignment_role_id),
```

Comment:
Actual bug was caused by this line. The assignee shown in the issue list would be `{type: 'role', id: 'None'}` because it would (wrongly) stringify `str(assignment_user_group_id)` when a role was set as the assignee